java/statestore/src/test/java/sleeper/statestore/s3/S3StateStoreIT.java

```diff
@@ -481,7 +481,7 @@ public void shouldAtomicallyUpdateStatusToReadyForGCAndCreateNewActiveFile() thr
     }
 
     @Test
-    public void shouldAtomicallyUpdateStatusToReadyForGCAndCreateNewActiveFilesForSplittingJob() throws Exception {
+    public void shouldAtomicallyUpdateStatusToReadyForGCAndCreateNewActiveFilesForFileSplitting() throws Exception {
```

Comment:
I'm not sure this is valid any more, as atomicallyUpdateFilesToReadyForGCAndCreateNewActiveFiles isn't used for splitting file references now.